.github/workflows/lint.yaml

```diff
@@ -26,6 +26,7 @@ jobs:
           # Enable linters
           black: true
           flake8: true
+          flake8_args: "--ignore=E501"
```

Comment:
I'd recommend using setup.cfg for it so that it will match your local linting environment: https://github.com/greenelab/ponyo/blob/master/setup.cfg